Tests/TensorFlowTests/LossTests.swift

```diff
@@ -217,6 +217,23 @@ final class LossTests: XCTestCase {
         assertEqual(computedGradient, expectedGradient, accuracy: 1e-6)
     }
 
+    func testHuberLoss() {
+        let predictions = Tensor<Float>([[0.9, 0.2, 0.2], 
+                                         [0.8, 0.4, 0.6]])
+        let targets = Tensor<Float>([[1, 0, 1], 
+                                     [1, 0, 0]])
+
+        // test huber(x, x) = 0
+        let loss1 = huberLoss(predicted: predictions, expected: predictions, delta: Float(1))
+        XCTAssertTrue(loss1.isAlmostEqual(to: Tensor(0), tolerance: 0.001), "Huber Loss all correct")
```

Comment:
Please use `assertEqual` instead, as other tests are using. This provides more information than `XCTAssertTrue`.

```suggestion
        assertEqual(loss1, Tensor(0), accuracy: 1e-6)
```

Same below for testing `loss2` please.